ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-release-4.14.yaml

```diff
@@ -342,7 +342,53 @@ tests:
     pre:
     - chain: ipi-aws-pre
     - chain: create-infra-move-ingress-monitoring-registry
-    workflow: openshift-qe-ocp-qe-perfscale-ci-tests
+    workflow: openshift-qe-cluster-density-v2
+  timeout: 5h0m0s
+- always_run: false
+  as: qe-perfscale-aws-ovn-small-cluster-density
+  steps:
```

Comment:
add `optional: true` to each job please.

Reply:
I missed that! Thanks for catching that.